tests/test_resolutions.py

```diff
@@ -198,7 +196,15 @@ def test_description(self) -> None:
 
         cti = ContainsTheItem(test_item)
 
-        expected_description = f'A sequence containing "{test_item}".'
+        expected_description = "A sequence containing <1>."
+        assert cti.describe() == expected_description
+
+    def test_description_str(self) -> None:
+        test_item = "1"
+
+        cti = ContainsTheItem(test_item)
+
+        expected_description = "A sequence containing '1'."
```

Comment:
I don't think these are useful tests to add, since we have the test of strings-vs-not-strings in `test_speech_tools` already. I think we should remove them.

("These tests" being all of the `test_description_str`s that have been added in this PR.)

Reply:
This is one of those times that a confluence of more than one test comes together to prove `ContainsTheItem` does the right thing.  In this case there is some overlap with the speechtools, but it also proves that the class utilizes the speechtools as expected.    I see no harm in some overlap.

Reply:
I can agree with that part. In that case though i'd rather that the test is more explicit about what it's testing, and tests both a number and a string:
```python
    def test_description_uses_represent_prop(self) -> None:
        str_cti = ContainsTheItem("1")
        int_cti = ContainsTheItem(1)

        assert str_cti.describe() == "A sequence containing '1'."
        assert int_cti.describe() == "A sequence containing <1>."
```

Reply:
I updated the tests to match your suggestion.  In doing so I noticed I completely missed the cases for greater & lessthan